tools/python/FindOptimizerOpsetVersionUpdatesRequired.py

```diff
@@ -59,18 +62,53 @@ def get_call_args_from_file(filename, function_or_declaration):
     return results
 
 
+def get_multiline_call_args_from_file(filename, function_or_declaration):
+    """
+    Search a file for all function calls or declarations that match the provided name.
+    Allows the opening '(' and closing ')' to be split across multiple lines.
+    Supports a single call per line.
+    """
+
+    results = []
+    with open(filename) as f:
+        cur_line = None
```

Comment:
If I understand correctly `cur_line` is the temp store for a multiline call? If that's the case would there be a better name that captures what the variable is? E.g. `current_function_call_or_declaration` etc?

I would also avoid abbreviating by deleting letters within a word.

https://google.github.io/styleguide/pyguide.html#316-naming
https://google.github.io/styleguide/cppguide.html#General_Naming_Rules